index.html

```diff
@@ -180,6 +180,61 @@ <h1 class="aboutmeh1">
                 <li>Web Development</li>
             </ul>
         </section>
-        
+        <section class="contactsection">
+            <div>
+                <h1>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h1>
+            </div>
+            <!-- <img class="line" src="img/Vector.png" alt="LineImage"> -->
+            <form action="https://formspree.io/f/xzbqjzwo" method="post">
+                <ul class="formlu">
+                  <li class="nameinputli">
+                    <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required/>
+                  </li>
+                  <span class="linespan"></span>
+                  <li class="emailinputli">
+                    <input type="email" id="mail" name="user_email" placeholder="Email address" required/>
+                  </li>
+                  <span class="linespan"></span>
+                  <li>
+                    <textarea id="msg" name="user_message" placeholder="Write me something..." maxlength="500" required></textarea>
+                  </li>
+                  <li class="submitform">
+                    <button type="submit">Get in touch</button>
+                  </li>
+                  
+                </ul>
+            </form>
```

Comment:
- Kindly remove the ul and li tags in the form. These are basically used as bullets to list items but in this case, you won't be needing them. The inputs field are enough